Makefile

```diff
@@ -349,3 +361,51 @@ integration-test:
 	else\
 		$(GINKGO) -v $(TEST_ARGS) --fail-fast --focus-file=$(TEST_FILE) test/integration;\
 	fi
+
+#hosted
+ADDON_CONTROLLER = $(PWD)/governance-policy-addon-controller
+MANAGED_CLUSTER_NAME_IN_HOSTED = cluster2
```

Comment:
I think you can just use `MANAGED_CLUSTER_NAME` directly instead if you stop overriding `MANAGED_CLUSTER_NAME`.

Reply:
I tested several times with default timeout, iam and cert policy test had fallen to fail... but set to 60secs working well. I guess hosted mode takes more time than normal one

Reply:
If you follow my comment about setting `MANAGED_CLUSTER_NAME` to `cluster2`, then we shouldn't need this new variable of `MANAGED_CLUSTER_NAME_IN_HOSTED`.

Reply:
I think you should still be able to use `MANAGED_CLUSTER_NAME` instead of this new variable.